mmv1/products/kms/CryptoKey.yaml

```
@@ -85,6 +85,34 @@ properties:
     name: 'labels'
     description: |
       Labels with user-defined metadata to apply to this resource.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'primary'
+    description: |
+      A copy of the primary CryptoKeyVersion that will be used by cryptoKeys.encrypt when this CryptoKey is given in EncryptRequest.name.
+      The CryptoKey's primary version can be updated via cryptoKeys.updatePrimaryVersion.
+      Keys with purpose ENCRYPT_DECRYPT may have a primary. For other keys, this field will be omitted.
```

Normally we try to stick as close as possible to API docs but we have to remove API-specific information or change anything that's different between the API and Terraform. If there's a specific non-obvious way to update the primary version in Terraform it would be great to include that here instead of just deleting the line.

```suggestion
      A copy of the primary CryptoKeyVersion that will be used by cryptoKeys.encrypt when this CryptoKey is given in EncryptRequest.name.
      Keys with purpose ENCRYPT_DECRYPT may have a primary. For other keys, this field will be unset.
```